### PR TITLE
Split Electrical.Machines into single files

### PR DESCRIPTION
**Pull request by AHaumer**
See #2975.

### CODE REVIEW

File: Modelica/Electrical/Machines/BasicMachines/Components/package.mo [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (beutlich) [inline Modelica/Electrical/Machines/BasicMachines/Components/package.mo]: No need to have redundant empty lines.

**Comment by christiankral** [comment]
Please consider https://github.com/AHaumer/ModelicaStandardLibrary/pull/6 as there was some leftover from a (merge conflict) in the code

**Comment by beutlich** [comment]
> 
> 
> There still is a conflict since the changes of [86c4774](https://github.com/modelica/ModelicaStandardLibrary/commit/86c4774e7364c769a7fbbe71921985e1be11a8db) are not considered in the PR.

Resolved.

**Comment by beutlich** [comment]
@dietmarw Can you please have a look at the new example model Modelica.Electrical.Machines.Examples.DCMachines.DC_CompareCharacteristics.

**Pull request merged**